### ISSUE TITLE
Creating a new cluster with custom config

### ISSUE THREAD

**Issue by ismferd** [kubernetes-sigs/kind#1335]
Hi,

I'm facing with:
```
Error: failed to create cluster: decoding failure: no kind "Cluster" is registered for version "kind.x-k8s.io/v1alpha4" in scheme "pkg/runtime/scheme.go:101"
```
I'm not sure if that is an issue or a misconfiguration, I'm trying to run kind executing `kind create cluster --config kind.yaml` and the config file contains:
```

# this config file contains all config fields with comments
# NOTE: this is not a particularly useful config file
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
# 1 control plane node and 3 workers
nodes:
# the control plane node config
- role: control-plane
# the three workers
- role: worker
- role: worker
- role: workers
```
am I doing something wrong?

**Comment by tao12345666333** [comment]
please show me your `kind --version`

**Comment by ismferd** [comment]
sure!
```
○ → kind --version
kind version v0.5.1
```

**Comment by ismferd** [comment]
ok, Upgrading version to 0.7.0 is woking as I expected.

Thanks @tao12345666333

**Comment by tao12345666333** [comment]
For detail:

`kind.x-k8s.io/v1alpha4` is only available since v0.6+

v0.5 uses another version of the configuration.
